wireless_modules/middle_module/mqtt_client.py

```diff
@@ -20,44 +19,38 @@ def __init__(self, client_id, broker_address):
     def connect_and_subscribe(self, topics_to_subscribe, callback_func):
         """
         Connects to the MQTT broker and subscribes to topic in 'topics_to_subscribe' using QoS = 1
-        :param ``topics_to_subscribe``: An array of topics to subscribe to.
+        :param topics_to_subscribe: An array of topics to subscribe to.
                                     Each element must be a string or byte literal (the latter is preferred)
         :param callback_func: The function to be called whenever a message from the subscribed topics is received.
-        :return: True if a connection to the Broker is successfully established, otherwise False
         """
 
         self.client.set_callback(callback_func)
-        try:
-            # Connect to MQTT broker
-            self.client.connect()
-            print('Connected to %s MQTT broker' % (self.mqtt_broker))
-        except OSError:
-            return False
+
+        # Connect to MQTT broker
+        self.client.connect()
+        print("Connected to %s MQTT broker" % self.mqtt_broker)
 
         # Subscribe to each topic
         for topic in topics_to_subscribe:
             self.client.subscribe(topic, qos=1)
-            print('Subscribed to %s topic' % (topic))
+            print("Subscribed to %s topic" % (topic))
 
         return True
 
     def _to_bytes_literal(self, data):
         """
-        Converts ```data``` into a form MQTT can read
+        Converts data into a form MQTT can read
         :param data: A string of data to convert to bytes literal
-        :return: The bytes literal version of the ```data```
+        :return: The bytes literal version of the 'data'
         """
         str_data = str(data)
         return str.encode(str_data)
 
     def mqtt_pub(self, data, topic):
```

Comment:
Just to neaten the API a little, could you put `topic` as the first parameter and then make `data` optional, defaulting to an empty string?